src/hooks/useLoginRedirectPath.ts

```diff
@@ -0,0 +1,25 @@
+import { authenticatedRoutes, REDIRECT_TO_PARAM_NAME } from 'app/routes';
+import { useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+
+// If no redirectToBase is passed in then we assume you want to get redirected to the home page
+function useLoginRedirectPath(redirectToBase?: string) {
```

Comment:
We had this logic in a few places so moved it into a hook.

Once change to the actual logic is now we default the redirect to the home page if nothing is there.